### PR TITLE
Migrated newComers page to  Layer5-ng

### LINKED ISSUE #1169: [NG] Migrate the Newcomers page
**Desired Situation**

Migrate the [Newcomers page](https://layer5.io/community/newcomers) from layer5.io to the Layer5-ng site.

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)

### PR DESCRIPTION
**Pull request by kishore007k**
Signed-off-by: kishore007k <kishorekumarstar07@gmail.com>

**Description**

This PR fixes #1169 

**Notes for Reviewers**
This PR is a draft, for now. Once all the changes are made will convert.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by Tanuj22** [comment]
@kishore007k is this ready for review?

**Comment by kishore007k** [comment]
> @kishore007k is this ready for review?

Yes, @Tanuj22

File: src/sections/Newcomers/NewcomersPageWrapper.style.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
+import styled from "styled-components";
+
+const NewcomersPageWrapper = styled.section`
+	max-width: 1170px;
+	display: block;
+	height: auto;
+	margin: 200px auto;
+
+	a {
```
Comment (Tanuj22) [inline src/sections/Newcomers/NewcomersPageWrapper.style.js:9]: Remove this. This will be done centrally after we decide on the colors for links

**Review by Tanuj22 — CHANGES_REQUESTED** [review]
Nice work. Some changes - 
- For both the tables can you follow the style on this page https://deploy-preview-1173--layer5ng.netlify.app/programs/hacktoberfest/2020
- For the tutorial's table can you keep the data in `assets/data` so that its easier to add a tutorial later on.

File: src/sections/Newcomers/NewcomersPageWrapper.style.js (hunk 1 above)
Comment (kishore007k) [reply src/sections/Newcomers/NewcomersPageWrapper.style.js:9]: Yeah, Sure I Will do. And one more thing. In the [layer5.io](https://layer5.io/community/newcomers) site we don't have the video I added it in this. Should I have to keep that or not?

Comment (Tanuj22) [reply src/sections/Newcomers/NewcomersPageWrapper.style.js:9]: You can remove it. Anyways it is not something we want to keep on the site

**Review by Tanuj22 — APPROVED** [review]
Good work @kishore007k. Thank You for working on this

**Pull request merged**